templates/delete.html

```diff
@@ -0,0 +1,37 @@
+{% extends "layout.html" %}
+{% block title %}To-Do App{% endblock %}
+
+{% block navbar %}
+<div class="collapse navbar-collapse" id="navbarText">
```

Comment:
When using `{% Jinja expression %}` I'd indent its content (i.e the `div` below) one level in

Reply:
Done